tests/target/pagination_dates/dates/2016/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>test</title>
+    </head>
+    <body>
+        <h1>dates.html</h1>
+
+        
+  
+  
+  <div>Date: 2016</div>
+  
+    <div>Sub-dates:</div>
+  
+
+
+All years:
+<ul>
+  
+    <li><a href="/dates/2016/1/">1 (1)</a>
```

Comment:
What about supporting zero-padding?  At least for me, I prefer it.

Reply:
I prefer it too, but that's why I asked about the `date` filter, I was counting on it to do that as in the code, it's all `u32`. Shall I try to do it in cobalt directly?

Reply:
`date` filter won't help with the paths being generated, that is purely in cobalt.

Reply:
True, I'll try to figure it out